### DISCUSSION TITLE
Removal of legacy precompiled model

### DISCUSSION THREAD

**Discussion by fabiocav** [Azure/Azure-Functions#1975] [Design discussions]
Discussion for proposal #1974

**Comment by brettsam** [comment]
Questions during meeting:
- What are the historical reasons for keeping this around? Are there major products relying on this behavior?
- Removing this could lose some "dynamic" host.json configuration scenarios -- for example, manually modifying values in host.json does not work for the attribute-based model.
- Sample of this is here -- https://github.com/Azure/azure-functions-host/wiki/Precompiled-functions
- Need investigation of subs currently using this support today.
- Would removing this result in us removing a "known" amount of support for some "unknown" amount of support with a new model. Would that be valuable work for the team?
- Can we add a detector for this instead?

We vote to reject this...
